yellowbrick/utils.py

```diff
@@ -155,6 +157,45 @@ def is_dataframe(obj):
 isdataframe = is_dataframe
 
 
+#From here: http://stackoverflow.com/questions/26248654/numpy-return-0-with-divide-by-zero
+def numpy_div0( a, b ):
+    """
+    Ufunc-extension that returns 0 instead of nan when dividing numpy arrays
+    example: >> div0( [-1, 0, 1], 0 ) -> [0, 0, 0]
+    """
+    with np.errstate(divide='ignore', invalid='ignore'):
+        c = np.true_divide( a, b )
+        c[ ~ np.isfinite( c )] = 0  # -inf inf NaN
+    return c
+
+
+def find_text_color(base_color, text_colors=("black","white")):
+    '''
+    Takes a background color and returns the appropriate light or dark text color.
+    Users can specify the dark and light text color, or accept the defaults of 'black' and 'white'
+
+    # baseColor: the color of the background, in any valid color specification format
+    # textColors: a tuple of (dark, light) text format. 
+    '''
+
+    #matplotlib cmap exports rgba colors scaled 0-1 instead of standard 255
+    rgb=[]
+    for idx,val in enumerate(base_color):
+        rgb.append(val*255)
+    
+    #Calculate brightness relative to the color type
+    base_brightness = math.sqrt( .241*rgb[0]*rgb[0]+
+                                 .691*rgb[1]*rgb[1]+ 
+                                 .068*rgb[2]*rgb[2] 
+                                ) 
+
+    if base_brightness > 130:
+        #base is light
+        return text_colors[0]
+    else:
```

Comment:
point of minor annoyance that I think would be best practice to help us out in open source land, convert:

```python 
if bool:
    return a 
else:
    return b 
```

to 

```python

if bool:
    return a 
return b 
```

The fear is that the else or conditional gets modified in such a way that return None becomes possible, which can break things in such a way so as to be very difficult to solve! 

If you don't want to change it, I won't fight you, but thought I'd comment since this relates to friendliness rather than style.